experimental/Schemes/src/ToricBlowups/constructors.jl

```diff
@@ -61,45 +64,52 @@ end
     blow_up(v::NormalToricVariety, new_ray::AbstractVector{<:IntegerUnion}; coordinate_name::String = "e")
 
 Blow up the toric variety by subdividing the fan of the variety with the
-provided new ray. Note that this ray must be a primitive element in the
-lattice Z^d, with d the dimension of the fan. This function returns the
-corresponding blowdown morphism.
+provided new ray. This function returns the corresponding morphism.
+
+Note that this ray must be a primitive element in the lattice Z^d that
+does not coincide with any existing ray, with d the dimension of the
+fan. In particular, it is currently impossible to blow up along a ray
+which corresponds to a non-Q-Cartier divisor.
```

Comment:
```suggestion
```
#3926 fixes this.

Reply:
Removed "does not coincide with any existing ray".